undebt/pattern/interface.py

```diff
@@ -78,10 +84,23 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        path = module_name_to_path(path)
```

Comment:
@ajm188 Was just poking around--what you actually want to do instead is use `importlib.import_module` combined with the `sys.path` trick, which should do everything you want, including handling modules not in the current directory.